TravisCI Build Failed For Release v0.0.1

## What happens?

Container images did not get built and pushed to quay.io when the v0.0.1 release was created.

## What were you expecting to happen?

I expected the container images to be built and pushed to quay.io with the tag v0.0.1.

## Steps to reproduce: 

Create a release in this repo.

## Any errors, stacktrace, logs? 

See details here ... https://travis-ci.com/KohlsTechnology/eunomia/jobs/229139174

```
Skipping a deployment with the script provider because this branch is not permitted: v0.0.1

1.75s$ rvm $(travis_internal_ruby) --fuzzy do ruby -S gem install dpl
Successfully installed dpl-1.10.12
1 gem installed

Installing deploy dependencies
Successfully installed dpl-script-1.10.12
1 gem installed

Preparing deploy

Deploying application
docker login -u [secure] -p [secure] quay.io
WARNING! Using --password via the CLI is insecure. Use --password-stdin.
WARNING! Your password will be stored unencrypted in /home/travis/.docker/config.json.
Configure a credential helper to remove this warning. See
https://docs.docker.com/engine/reference/commandline/login/#credentials-store
Login Succeeded
./scripts/build-images.sh quay.io/kohlstechnology
/usr/bin/env: ‘bash -e’: No such file or directory
Makefile:53: recipe for target 'travis-deploy-images' failed
make: *** [travis-deploy-images] Error 127
Script failed with status 2
failed to deploy
```

## Environment:

* Runtime version(Java, Go, Python, etc): n/a
* Desktop OS/version: n/a

## Additional comments:

n/a

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,0 +1,8 @@
+## v0.0.2 (Unreleased)
+This release has the same content as release v0.0.1. A bug
+was found in the Travis CI automation that prevented the v0.0.1
+container images from being pushed to quay.io.
+
+## v0.0.1 (August 28, 2019)
+* Initial release
+* Dead on arrival, do not use

--- a/scripts/build-images.sh
+++ b/scripts/build-images.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash -e
+#!/usr/bin/env bash
 
 # Copyright 2019 Kohl's Department Stores, Inc.
 #
@@ -13,6 +13,8 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+set -e
 
 REPOSITORY=${1}
 if [ -z "${TRAVIS_TAG}" ] ; then

